include/cudaq/Optimizer/Dialect/Quake/QuakeOps.td

```diff
@@ -597,22 +699,26 @@ class TwoTargetOp<string mnemonic, list<Trait> traits = []> :
                 traits)>;
 
 //===----------------------------------------------------------------------===//
-// Pauli instructions
+// Quantum operators (gates)
 //===----------------------------------------------------------------------===//
 
-def XOp : OneTargetOp<"x", [Hermitian]> {
-  let summary = "Pauli-X operation (aka, NOT)";
+def HOp : OneTargetOp<"h", [Hermitian]> {
+  let summary = "Hadamard operation";
   let description = [{
+    This is a pi rotation about the X+Z axis, and has the effect of changing
```

Comment:
update to "changing the computational basis state from..."

Reply:
Also I notice that pi is written with the greek symbol in other places, maybe update that here too